src/Groups/homomorphisms.jl

```diff
@@ -386,22 +386,27 @@ function hom(x::GAPGroupElem{AutomorphismGroup{T}}) where T <: GAPGroup
   return _hom_from_gap_map(G, G, x.X)
 end
 
-(f::GAPGroupElem{AutomorphismGroup{T}})(x::GAPGroupElem{T}) where T <: GAPGroup = apply_automorphism(f, x)
-Base.:^(x::GAPGroupElem{T},f::GAPGroupElem{AutomorphismGroup{T}}) where T <: GAPGroup = apply_automorphism(f, x)
+(f::GAPGroupElem{AutomorphismGroup{T}})(x::GAPGroupElem) where T <: GAPGroup = apply_automorphism(f, x)
+Base.:^(x::GAPGroupElem,f::GAPGroupElem{AutomorphismGroup{T}}) where T <: GAPGroup = apply_automorphism(f, x)
+Base.:^(f::GAPGroupElem{AutomorphismGroup{T}},g::GAPGroupElem{AutomorphismGroup{T}}) where T <: GAPGroup = g^-1*f*g
```

Comment:
I think the only reason you need this line above is because our generic definition for conjugation is unnecessarily ambiguous. It currently says this:
```julia
Base.:^(x::GAPGroupElem, y::GAPGroupElem) = ...
```
I think if you change that as follows, the above line may not be needed:
```julia
Base.:^(x::T, y::T) where {T <: GAPGroupElem} = ...
```

Reply:
It's needed anyway, because the operation `x^f`, intended as the evaluation of the automorphism `f` in the variable `x`, isn't defined anywhere else.

Reply:
Sure, that's the one in the line *above*, i.e., `Base.:^(x::GAPGroupElem,f::GAPGroupElem{AutomorphismGroup{T}})`.

But I am talking about `Base.:^(f::GAPGroupElem{AutomorphismGroup{T}},g::GAPGroupElem{AutomorphismGroup{T}})`

Reply:
Ping?

Reply:
Done it.